src/App.module.scss

```diff
@@ -1,3 +1,23 @@
+@import "styles/variables";
+
 .app {
-  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  background-color: $background-color;
+  font-family: $font-family;
```

Comment:
I suggest setting the default font family at the root level (index.css maybe), and override it when necessary.

Reply:
yeah, good call.